cmd/eip_show.go

```diff
@@ -21,6 +21,7 @@ type eipHealthcheckShowOutput struct {
 }
 
 type eipShowOutput struct {
+	Description string                    `json:"description"`
```

Comment:
Can you please move the `Description` field after the `IPAddress` one? 😬

Reply:
done

Reply:
done